mitmproxy/io/har.py

```diff
@@ -109,7 +120,7 @@ def request_to_flow(request_json: dict) -> http.HTTPFlow:
 
     # Then encode the content, as in `Response.set_content`
     response_content = http.encoding.encode(
-        response_content, response_headers.get("content-encoding") or "identity"
+        response_content, fix_content_encoding(response_headers.get("content_encoding"))
     )
```

Comment:
I'd like us to be a bit more Pythonic here and ask for forgiveness rather than for permission. Can we just do something like this please?

```python3
    try:
        # Then encode the content, as in `Response.set_content`
        response_content = http.encoding.encode(
            response_content, response_headers.get("content-encoding") or "identity"
        )
    except ValueError:
        logger.warn(f"HAR import: Ignoring invalid content-encoding in request to {request_url!r}")
```